CONTRIBUTING.md

```diff
@@ -162,11 +162,11 @@ $ ./rebar3 ct
 ```
 
 Most tests are named according to their module name followed by the `_SUITE`
-suffi. Providers are made shorter, such that `rebar_prv_new` is tested in
+suffix. Providers are made shorter, such that `rebar_prv_new` is tested in
 `rebar_new_SUITE`.
 
 Most tests in the test suite will rely on calling Rebar3 in its API form,
-then investigating the build output. Because most tests have similar
+then investigate the build output. Because most tests have similar
```

Comment:
'investigating' is correct there since we are still discussing what the tests will rely on: 
- calling ...
- investigating ...